command/server.go

```diff
@@ -828,14 +827,14 @@ type quiescenceSink struct {
 	t *time.Timer
 }
 
-func (q quiescenceSink) Accept(name string, level log.Level, msg string, args ...interface{}) {
+func (q quiescenceSink) Accept(name string, level hclog.Level, msg string, args ...interface{}) {
 	q.t.Reset(100 * time.Millisecond)
 }
 
 func (c *ServerCommand) setupStorage(config *server.Config) (physical.Backend, error) {
 	// Ensure that a backend is provided
 	if config.Storage == nil {
-		return nil, fmt.Errorf("A storage backend must be specified")
+		return nil, errors.New("A storage backend must be specified")
```

Comment:
I prefer to use `fmt.Errorf` over `errors.New`, except in the case where I'm declaring a variable to hold an error which I will compare against in code to see if an error is of a particular type, since this is a widespread convention.  Rationale: consistency, as most returned errors will include parameters, why have a second way of returning an error that's only very rarely used?

Reply:
Consistency is important. In my opinion, `fmt.Errorf` is useful when these are other args to include, as this function call `errors.New` if no args provided. 
I found this file that include both `fmt.Errorf` and `errors.New` usages: https://github.com/mayocream/vault/blob/0ef61459b82d1c4287e3398540c0e3114fc2a3c8/builtin/logical/ssh/path_sign.go#L310
This could be different contributors using their way to return an error.

Reply:
I would reject this change as gratuitous if it were on its own - since both ways of returning a parameter-less error are equally valid, and it largely comes down to taste.  I wouldn't want to flip-flop between styles based on who the latest author is.  However, since there's a lot of other good stuff in this PR I'll allow it.

In future if I can get consensus internally for either approach (i.e. either "always use errors.New when there are no parameters" or "always use fmt.Errorf regardless") I'll look at adding a semgrep check to enforce that style.  Like with gofmt, it's better if there's only one way to do it so that people don't have to choose.